src/turtle/runner/Runner.d

```diff
@@ -506,13 +497,18 @@ class TurtleRunnerTask ( TestedAppKind Kind ) : TaskWith!(ExceptionForwarding)
             this.createSandbox();
         }
 
+        static if (AutoStartTestedApp)
+        {
+            this.createTestedApplication();
+        }
+
         // user hook for starting services / processes
         this.prepare();
 
-        static if (AutoStartTestedApp)
+        static if (AutoStartTestedApp && Kind == TestedAppKind.Daemon)
```

Comment:
I don't follow why `Kind` is relevant here ... ?  And what will happen if `Kind != Daemon` ... ?

Reply:
CLI apps get started manually inside test cases as they are run in blocking manner. Daemon app is started automatically and keeps running in the background.

Reply:
Oh, sorry.  Missed the fact that for `CliApp` the app will be started from test cases.  But it might be worth adding the `else` block with comment about the `CliApp` case.